src/components/app/details/appDetails/IssuesCard.tsx

```diff
@@ -22,7 +22,7 @@ import IndexStore from '../../../v2/appDetails/index.store'
 import { renderErrorHeaderMessage } from '../../../common/error/error.utils'
 import LoadingCard from './LoadingCard'
 
-const IssuesCard = ({ appStreamData, cardLoading, setErrorsList, toggleIssuesModal, setDetailed }: IssuesCardType) => {
+const IssuesCard = ({ appConditions, cardLoading, setErrorsList, toggleIssuesModal, setDetailed }: IssuesCardType) => {
```

Comment:
we can remove this prop, it can be fetched from appDetails

Reply:
done